api/src/database/index.ts

```diff
@@ -136,8 +136,9 @@ export function getDatabase(): Knex {
 	}
 
 	if (client === 'mysql') {
+		if (isObject(knexConfig.connection)) delete knexConfig.connection['filename'];
```

Comment:
`isObject` needs to be imported from `@directus/utils`

Reply:
Let's also add a small comment:
```suggestion
		// Remove the conflicting `filename` option, defined by default in the Docker Image
		if (isObject(knexConfig.connection)) delete knexConfig.connection['filename'];
```